blindml/frontend/config/task/task.py

```diff
@@ -62,9 +64,20 @@ def __init__(self, task_fp) -> None:
         self._experiment_name = f"{self.user}s_experiment"
         self._experiment_name_with_hash = f"{self._experiment_name}_{self._task_hash}"
         if self._data_path.endswith(".csv"):
+            print("Loading CSV")
+            (a,b) = os.path.split(self._data_path)
+
+            extra_path = a + "/" "extra_" + b
+            
+            if(os.path.exists(extra_path)):
+                print("Searching for related data")
+                time.sleep(10)
+                print("Found related data")
+                self._data_path = extra_path
             all_columns = next(
                 csv.reader(open(self._data_path, "r", encoding="utf-8-sig"))
             )
+            time.sleep(10)
```

Comment:
don't think we need this one...

Reply:
it was a specific request for the demo to make it seem like it was "doing something"

Reply:
this one makes sense for producing that effect

https://github.com/TheDataStation/blindml/pull/3/files/118ed3e05011d9e7b94979e17c884fe14c1a5aed#diff-b200ade7998b694ed509c2463e2d0ddcb64feb9c6ee092bbe85d78382a4ebbafR74

the second one does not as the library actually does do something with the data once it's been "discovered" ie trains models